crane-cloud/backend
# Update kubernetes version and fix Ingress

# Description
This PR updates the Kubernetes client version from `10` to `23`.
It also fixes the ingress configurations to use **NetworkingAPI** instead of the **ExtentsionsAPI**


## Trello Ticket ID
Link to Trello ticket is [here](https://trello.com/c/mJDgaC8d/1028-fix-ingress-deployment)

## How Can This Be Tested?

**Note:** the update does not work on staging (CIS- cluster), because it has Kubernetes version `v1.18`, but works on all the other clusters we have at the moment.

- Create a project and an app on the other clusters and it should work.

## Checklist:
